zebra/rt_netlink.c

```diff
@@ -105,7 +105,7 @@ struct gw_family_t {
 };
 
 static const char ipv4_ll_buf[16] = "169.254.0.1";
-static struct in_addr ipv4_ll;
+struct in_addr ipv4_ll;
```

Comment:
I agree with Donald about this change - in fact the whole purpose of these two variables is really just to get four octets into 'ipv4_ll', and that could be accomplished more centrally and simply. maybe moving a variable to the lib/ipaddr module would clean this up?

Reply:
will update the code